LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Chain, Inc.
+Copyright (c) 2022, MaidSafe
```

Comment:
see https://gist.github.com/fbaierl/1d740a7925a6e0e608824eb27a429370

Don't remove the original copyright owner. Add Maidsafe underneath as an additional copyright owner

Reply:
Ah ok, thanks. I didn't anticipate having multiple attributions in the licensing tool, so I'll need to change it to accommodate that. Will probably come back to it on Friday.

Reply:
The original copyright holder has been retained, and I extended the licensing tool to support multiple attributions. We need this PR merged: https://github.com/maidsafe/verify-licensing-info/pull/3

After this I should be able to re-run the failed job and it should pass. I've tested it on my fork.

Reply:
That's now been merged, so the check is passing.